MatrixSDK/JSONModels/MXEvent.h

```diff
@@ -211,6 +211,7 @@ FOUNDATION_EXPORT NSString *const kMXEventRelationRelatesToKey;
 FOUNDATION_EXPORT NSString *const MXEventRelationTypeAnnotation;    // Reactions
 FOUNDATION_EXPORT NSString *const MXEventRelationTypeReference;     // Reply
 FOUNDATION_EXPORT NSString *const MXEventRelationTypeReplace;       // Edition
+FOUNDATION_EXPORT NSString *const kMXMessageContentKeyNewContent;   // Edited content key
```

Comment:
Not sure to be honest, relationTypes are the only ones left on `MXEvent`, everything else is `kMXEvent`. I only placed it near them because it's directly related to `m.replace`

Reply:
Gotcha.